v1.1.1/index.html

```diff
@@ -38,7 +38,7 @@
 
 	<!-- Loopy & Ink -->
 	<div id="canvasses"></div>
-	<div id="graph_canvas"></div>
+	<div id="graph_canvas", style="width:400px"></div>
```

Comment:
Nitpick: The `,` isn't necessary in HTML, and in fact causes the HTML to come out as 
![image](https://user-images.githubusercontent.com/44409145/145431648-4914ade7-ef64-415b-af6a-dc2103dc37f9.png)

Reply:
Also, I've just been doing the style stuff in `loopy.css` since all the other styling stuff has been done there. Rather than having it in the HTML tags, you'd be better off just modifying line 20 of loopy.css